docker-app/qfieldcloud/core/migrations/0073_project_packaging_offliner.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
                     ("qgiscore", "QGIS Core Offline Editing (deprecated)"),
                     ("pythonmini", "Optimized Packager"),
                 ],
-                default="qgiscore",
+                default="pythonmini",
```

Comment:
Instead of editing an already existing migration, please create a new migration with this change only.